templates/details/publisher.html

```diff
@@ -33,4 +33,25 @@ <h1 class="p-heading--2">{{ publisher["display-name"] }}</h1>
   </div>
 </div>
 
+{% endblock %}
+
+{% block page_scripts %}
+
+<script>
+(function(doc) {
+  doc.addEventListener("DOMContentLoaded", function() {
+      // this function runs when the DOM is ready, i.e. when the document has been parsed
+      const cards = doc.getElementById("main-content").getElementsByClassName("p-card");
+      for (const card of cards) {
+        card.addEventListener("click", (event) => {
+          const link = card.querySelector("a");
+          if (link) {
+            link.click();
+          }
+        });
+      }
+  });
+})(document);
```

Comment:
Small nitpick so feel free to reject this one, `document` is a global so I find that passing it as an argument is superfluous.
```suggestion
(function() {
  document.addEventListener("DOMContentLoaded", function() {
      // this function runs when the DOM is ready, i.e. when the document has been parsed
      const cards = document.getElementById("main-content").getElementsByClassName("p-card");
      for (const card of cards) {
        card.addEventListener("click", (event) => {
          const link = card.querySelector("a");
          if (link) {
            link.click();
          }
        });
      }
  });
})();
```